certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -713,519 +428,1189 @@
+                    "description": "The identifier of the business entity type.",
                     "title": "Id"
                   },
                   "name": {
                     "type": "string",
-                    "description": "The name of the business entity type.",
+                    "description": "The business entity type name.",
                     "title": "Name"
                   },
                   "description": {
                     "type": "string",
                     "description": "The description of the business entity type.",
                     "title": "Description"
                   },
-                  "title": {
+                  "provider": {
                     "type": "object",
                     "properties": {
-                      "format": {
+                      "referenceId": {
                         "type": "string",
-                        "description": "format",
-                        "title": "",
-                        "x-ms-visibility": "internal"
+                        "description": "The reference identifier of the business entity connector.",
+                        "title": "Provider reference id"
                       },
-                      "fields": {
-                        "type": "array",
-                        "items": {
-                          "type": "string"
+                      "externalType": {
+                        "type": "string",
+                        "description": "The type that is used to identify the business entity type in an external system.",
+                        "title": "External Type"
+                      },
+                      "externalIdField": {
+                        "type": "string",
+                        "description": "Represents the field that is used in the external system to identify a business entity.",
+                        "title": "External identifier field"
+                      }
+                    },
+                    "description": "provider"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "List business entity provider types",
+        "description": "Lists all business entity types in the model",
+        "operationId": "List-Business-Entity-Provider-Types",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "providerReferenceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/api/Content": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK - Created",
+            "schema": {}
+          },
+          "201": {
+            "description": "OK - Created",
+            "schema": {}
```

Comment:
@JoepKillaars Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.